src/collections/lookup-map.ts

```diff
@@ -1,45 +1,45 @@
 import * as near from '../api'
-import { Bytes } from '../utils';
+import { u8ArrayToLatin1 } from '../utils';
 
 export class LookupMap {
-    readonly keyPrefix: Bytes;
+    readonly keyPrefix: string;
 
-    constructor(keyPrefix: Bytes) {
+    constructor(keyPrefix: string) {
```

Comment:
Why doesn't this accept `Bytes` input? Wouldn't it make more sense to store the prefix at `UInt8Array` since it isn't really a utf16 string but bytes (or intended to be)?

Reply:
Yeah, that makes most sense. I'm experimenting different approaches in collections, this one tries to keep backward compatibility but implementation then looks really awkward and not correct

Reply:
Isn't it backwards compatible if using `Bytes`? One of the variants would be string? Does it affect anything if the internal type changes?

Reply:
Yes it is backward compatible, if Bytes is string | Uint8Array.